final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import feedparser
+import os
+import pdfkit
+import random
+import dominate
+from dominate.tags import *
+from datetime import date
+
+import parse_news
+import get_cache
+import logger
+import version
+import converter
+
+
+arguments = argparse.ArgumentParser(description='Pure Python command-line RSS reader')
+
+arguments.add_argument('source', action='store', help='RSS URL')
+arguments.add_argument('--version', action='store_true', help='Print version info')
+arguments.add_argument('--json', action='store_true', help='Print result as JSON in stdout')
+arguments.add_argument('--verbose', action='store_true', help='Outputs verbose')
+arguments.add_argument('--limit', action='store', help='Limit news topics')
+arguments.add_argument('--date', action='store', help='Print news from the specified day')
+arguments.add_argument('--tohtml', action='store', help='Convert news in html format')
+arguments.add_argument('--topdf', action='store', help='Convert news in pdf format')
+
+args = arguments.parse_args()
+
+try:
+    # set object of feedparser library
+    channel = feedparser.parse(args.source)
+    print("Feed: ", channel.feed.title, '\n')
+    
+    # if no limit - take all news in channel
+    if not (args.limit):
+        args.limit = len(channel.entries)
+    limit = int(args.limit)
```

Comment:
It is better to specify argument type while parsing it. Check `type=` parameter for `add_argument` method

Reply:
Fixed